Contrast-Security-OSS/vulneruby_engine
# RUBY-1280/1281-Added-Thin-Max-and-Min-versions

### Changes Introduced  :sparkles:
This PR adds Maximum and minimum version for `Thin` in `AppTest`.

### How To Test :alembic:
1.) Checkout + build this branch

2.) Install in X application and visit X route

3.) Verify it works by:

4.) :white_check_mark:

### Additional Details :speech_balloon:
[RUBY-1280](https://contrast.atlassian.net/browse/RUBY-1280)
[RUBY-1281](https://contrast.atlassian.net/browse/RUBY-1281)
